main.go

```diff
@@ -186,6 +186,10 @@ func main() {
 	}
 
 	dbConnection := runMemphis(s)
+	err = s.RelaunchDlsForOldStations()
```

Comment:
change to launchDlsForOldStations() and add comment says that it meant for backward compatability